README.md

```diff
@@ -1,3 +1,45 @@
-# Futbol
+# Futbul
 
-Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+## Project Description
+
+
+For the Futbol project, we will be utilizing data from a fictional soccer
+league to analyze team performance across specific seasons and across all
+seasons. This will be done for individual teams and the league as a whole.
+
+
+## Organization
+
+
+The Futbol project was organized in a manner in which one class is not
+responsible for all of the methods. We broke up the methods based off of what
+it was evaluating. We created Game Statistics, League Statistics, Season
+Statistics and Team Statistics. Each of these classes are responsible for
+evaluating information from the given CSV files. Our StatTracker class is solely
+responsible for executing the methods that were created.
+
+
+## Testing
+
+
+Testing was conducted with different approaches. For the development of our
+methods, we utilized fixtures, the fixtures were created by importing data
+from the Game, GameTeams and Teams CSV files. This allowed our testing to
+perform its basic function which is to determine if the method gave us the
+information we needed. Using fixture files also allowed the testint times to
```

Comment:
testing*

Reply:
fixed the spelling. Let me know if you find any other typos.